scripts/jira-automation.py

```diff
@@ -140,7 +232,6 @@ def create_jira_issues(
             print(
                 f"dry run. No issues created. ({len(jira_issues_to_create)} issues would be created)"
```

Comment:
in the dry run option also add the details about the jiras to be created.

Reply:
I can loop the title maybe, or the entire dict?

Reply:
well, at least a description with the issue that is being created.

Reply:
sure got it

Reply:
added the `summary` which is concise yet provides the details well. Something like:
Snyk - [CVE-2023-44487] - [CWE-400] - [high] - [main] - red-hat-data-services/model-registry-operator - go.mod - Denial of Service (DoS)